### DISCUSSION TITLE
Remove focus from q-select after selection made

### DISCUSSION THREAD

**Discussion by cdecinkoKnight** [quasarframework/quasar#16065] [General - Components / Directives / etc] [answered]
Currently when a user makes a selection from a q-select drop down, the control highlights with a bold blue border. That's fine while interacting with the control. But, once they have selected something in the list, the border stays. How can I make it blur or lose focus after selecting?

**Comment by suleimanas** [comment, marked as answer]
by calling blur() method on @popup-hide event

- just an idea -

`<q-select ref="select_input" v-model="model" :options="options" label="Standard" @popup-hide="onSelectHide"></q-select>`


```
const select_input = ref();
    
function onSelectHide() {
  select_input.value.blur();
}
```

**Comment by cdecinkoKnight** [comment]
TypeError: Cannot read properties of null (reading 'type') among other warnings.

**Reply by suleimanas** [reply]
how did you do it, make a codepen so I can help you

**Reply by cdecinkoKnight** [reply]
It works now. I wasn't returning the ref. It compiled but failed during runtime. Works great. Thanks.
